### PR TITLE
amp-script: Share size limit across elements

### PR DESCRIPTION
**Pull request by choumx**
/to @dvoytenko /cc @kristoferbaxter

### CODE REVIEW

File: extensions/amp-script/0.1/amp-script.js [hunk 1]
```diff
@@ -109,12 +116,16 @@ export class AmpScript extends AMP.BaseElement {
       // `workerUrl` is from CDN, so no need for `ampCors`.
       xhr.fetchText(workerUrl, {ampCors: false}).then(r => r.text()),
       xhr.fetchText(authorUrl).then(r => r.text()),
+      getElementServiceForDoc(this.element, TAG, TAG),
```
Comment (jridgewell) [inline extensions/amp-script/0.1/amp-script.js]: Shouldn't this be on the `Services` object?

File: extensions/amp-script/0.1/amp-script.js [hunk 2]
```diff
@@ -194,6 +205,28 @@ export class AmpScript extends AMP.BaseElement {
...
+    /** @private {number} */
+    this.cumulativeSize_ = 0;
+  }
+
+  /**
+   * @param {number} size
+   * @return {boolean}
+   */
+  sizeLimitExceeded(size) {
+    this.cumulativeSize_ += size;
```
Comment (jridgewell) [inline extensions/amp-script/0.1/amp-script.js]: Nit: only add if less than max size.

File: extensions/amp-script/0.1/amp-script.js (hunk 1 above)
Comment (choumx) [reply extensions/amp-script/0.1/amp-script.js]: Didn't want to expose it since it's basically a "private" service.

**Comment by choumx** [comment]
Thanks for the quick review.

**Pull request merged**